__tests__/app.test.js

```diff
@@ -62,3 +62,77 @@ describe("/api/topics", () => {
     });
 });
 
+describe("/api/articles", () => {
+    describe("GET", () => {
+        test("Returns 'Status: 200' if no error in path", () => {
+            return request(app).get('/api/articles').expect(200);
+        });
```

Comment:
This test is unnecessary, as you will be expecting '200' for the status on the next test :)